.github/workflows/setup-anvil.yml

```diff
@@ -19,8 +19,22 @@ jobs:
           version: "latest"
       - name: Tunnel Anvil
         id: tunnel
-        run: cloudflared tunnel --url http://localhost:8545 &
+        run: cloudflared tunnel --url http://localhost:8545 --logfile cloudflared.log &
         shell: bash
+      - name: Extract Cloudflared URL
+        run: grep -E "https://.*.trycloudflare.com" cloudflared.log | cut -d'|' -f2 | tr -d ' ' > cloudflare-url.txt
```

Comment:
We should print the extracted URL to the logs

Reply:
done